core/domain/suggestion_registry.py

```diff
@@ -678,6 +682,24 @@ def accept(self, unused_commit_message):
         question_dict['version'] = 1
         question_dict['id'] = (
             question_services.get_new_question_id())
+        html_list = self.get_all_html_content_strings()
```

Comment:
Move this to a common-place maybe in the Base class? [We are going to have the same structure for other suggestions.

Reply:
We cannot move it to base class because it is different for each suggestion type.
For example for question suggestions we have to copy the images to the entity type question and similarly its different for translation type.